README.md

```diff
@@ -1,38 +1,74 @@
 # IWD dev team ladder
 
+This job ladder exposes all roles inside IWD development department, and define the goals, expectations and responsibilities of each role.
+Constant progression, and meeting your individual objectives will help you to climb it.
+
+Be aware that some basic expectations are not specified in the roles:
+* Being involved into team reviews
+* Delivering software on-time
```

Comment:
For me that's part of a progression path. For E1 you have to learn how to deliver on time. For E2 it should be ok.

Reply:
Fixed